theano/sandbox/gpuarray/type.py

```diff
@@ -751,3 +758,34 @@ def c_code_cache_version(self):
 declaration of an operation.
 """
 gpu_context_type = GpuContextType()
+
+
+# THIS WORKS But CudaNdarray instances don't compare equal to one
```

Comment:
I'm not sure this is still a useful comment. I don't want to go further in this PR as this work and leave Theano in a better state.

If you think it is important to understand if this comment is still true and investigate, we should open an issue about this. But I would consider this as non important as it work and I don't see a problem with this. But I could be wrong.

I updated the PR and made it skip travis that is overloaded.